test/signatures/ECDSA.t.sol

```diff
@@ -101,6 +109,143 @@ contract ECDSATest is Test {
         wrapper.verify(pk, m, sig);
     }
 
+    struct ECDSAValidCase {
+        string d;
+        string m;
+        string signature;
+    }
+
+    function testVectorsNobleCurves_verify_WithPublicKey() public {
```

Comment:
There is no test with `verify_WithAddress`, only `verify_WithPublicKey` for now. 
It should be equivalent so I think this is enough but happy to add more tests / make changes if necessary.

Reply:
Yeah it should be equivalent. Don't think the extra tests justify the code duplication.